lighthouse-core/audits/accessibility/aria-text.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Ensures `role=text` is only used on elements with no focusable descendants.
```

Comment:
Rob seemed like he was lukewarm about some of the possible `aria-text` gotchas and it's a `best-practice`, not `wcag2a` or `wcag2aa`. Should we drop for now?

Reply:
rob said:

> aria-text is kind of news to me and looks like it has a potential gotcha (like if you have a link inside of the heading) so...hm...idk

i said:

> apparently aria-label is the role to use if element has focusable elements, not aria-text. altho TIL role=text isnt standard yet? https://incl.ca/roletext-presently-kinda-not-thing-sorta/#:~:text=here-,role%3D%22text%22%20isn,While,-that

or are you referring to something else?

`role=text` not being standard seems like a convincing reason to drop/hide/0-weight it.

Reply:
yes, that's what I was trying to summarize without using many braincells, apparently :)